doc/ssh_config/ssh_config.md

```diff
@@ -80,6 +86,8 @@ We want to enable configuring the following policies, with default values are ta
 | authentication retries |     Number of attempts to try to log in   before rejecting the session    |     3-100               |     6            |
 | login timeout  |     SSH session timeout                                                   |     1-600 (secs)        |     120          |
 | ports          |     Port numbers for SSH                                                  |     1-65535             |     22           |
+| auto logout    |     Inactivity timeout for SSH session                                    |     0-35000 (min)       |     15           |
+| max sessions   |     Max number of concurrent logins                                       |     3-100               |     100          |

```

Comment:
Currently we already have template files and hostcfgd code to support generate session limit by hwsku:

https://github.com/sonic-net/sonic-buildimage/pull/10177

In the public repo this is a empty template.

So for the implementation detail of these limit, please consider reuse and improve existed code and template.

Reply:
Also for the auto logout, currently there are code in this file do some pre-configurition:

https://github.com/sonic-net/sonic-buildimage/blob/master/build_debian.sh

So the implementation should not break those code, or need migrate those config to the new design.

Reply:
Agree, we will reuse existing infra to keep all limits.conf file updates in one pllace. 
Added section to clarify this. Thank you for commens.